# Subscriber
car_state_topic_name: /estimation/slam/state

transform_matrix_topic_name: /transform_matrix

# Publisher
path_generate_topic_name: /planning/ref_path

ref_path_topic_name: /visual/ref_path

# Trajectory params:
desire_vel: 3.0

car_length: 1.88

look_ahead: 20

interval: 0.05

forward_distance: 15.0

circle_radius: 9.125

max_lat_acc: 3.0



# other params:
N: 40

dt: 0.04

initial_velocity: 2

mission: skidpad          # [trackdrive, skidpad, acceleration]

simulation: true

node_rate: 100   # [Herz]
